kernel/kernel-util/src/main/java/org/sakaiproject/util/EmailNotification.java

```diff
@@ -645,6 +645,24 @@ protected int getOption(User user, String notificationId, String resourceFilter,
 		String type = getType(resourceFilter);
 		if (type != null)
 		{
+            // First, check the overrides.
```

Comment:
I don't think there's an official policy (?), but my personal opinion is that code should match the existing style of the file.

I know this isn't the most important thing.

When doing a pull request you are normally ok todo a forced push to your local branch, that way you can keep using the existing pull request.